mopidy_listenbrainz/ext.conf

```diff
@@ -2,3 +2,4 @@
 enabled = true
 token = 
 url = api.listenbrainz.org
+import_playlists = false
```

Comment:
Do we need a rule for the update playlists item here?

Reply:
This configuration key has been removed.